server/map_rpc_server.go

```diff
@@ -321,6 +306,57 @@ func (t *TrillianMapServer) SetLeaves(ctx context.Context, req *trillian.SetMapL
 	return &trillian.SetMapLeavesResponse{MapRoot: newRoot}, nil
 }
 
+func doPreload(ctx context.Context, tx storage.MapTreeTX, hkv []merkle.HashKeyValue) error {
```

Comment:
For the ease of understanding for future readers, I recommend documenting briefly what this method is going to preload, and where things will be preloaded to.

Reply:
👍 done